src/test/java/uk/gov/hmcts/sptribs/common/notification/CancelHearingNotificationTest.java

```diff
@@ -165,12 +188,17 @@ void shouldNotifyApplicantWithEmail() {
 
         //Then
         verify(notificationService).sendEmail(any(NotificationRequest.class));
+        verify(notificationHelper).buildEmailNotificationRequest(
+            "testapplicant@outlook.com",
```

Comment:
Could you make this consistent to the other tests by changing to data.getCicCase().getApplicantEmailAddress()